### PR TITLE
Add Aalborg University to CS ranking

### PR DESCRIPTION
**Pull request by hujilin1229**
Here is an update for Department of Computer Science at Aalborg University, Denmark. In this update I have included almost all professors and associate professors in csrankings.csv. Since our department is not in this ranking before, I have included "Aalborg University" in country-info.csv.

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -16153,3 +16153,37 @@ François Fleuret,EPFL,https://people.epfl.ch/francois.fleuret,Bj1tRlsAAAAJ
 Alexandre Alahi,EPFL,https://people.epfl.ch/alexandre.alahi,UIhXQ64AAAAJ
 Pierre Vandergheynst,EPFL,https://people.epfl.ch/pierre.vandergheynst,1p9NOFEAAAAJ
 Ali H. Sayed,EPFL,https://people.epfl.ch/ali.sayed,uDG9sXQAAAAJ
+Christian S. Jensen,Aalborg University,http://people.cs.aau.dk/~csj/,wo20cPYAAAAJ
+Kim G. Larsen,Aalborg University,www.cs.aau.dk/~kgl,neDFD60AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:16157]: please make a proper URL

File: csrankings.csv [hunk 2]
```diff
@@ -16153,3 +16153,37 @@ François Fleuret,EPFL,https://people.epfl.ch/francois.fleuret,Bj1tRlsAAAAJ
...
+Hans Hüttel,Aalborg University,http://people.cs.aau.dk/~hans/index-eng.html,Wf7-agQAAAAJ
+René Rydhof Hansen,Aalborg University,http://people.cs.aau.dk/~rrh/,MGLEvwkAAAAJ
+Ulrik Nyman,Aalborg University,http://ulrik.blog.aau.dk/,Z3FXuI0AAAAJ
+Christian Thomsen,Aalborg University,http://personprofil.aau.dk/111293?lang=en,Y7i1yN4AAAAJ
+Lone Leth Thomsen,Aalborg University,http://people.cs.aau.dk/~lone/,vyYcihcAAAAJ
+Ivan Aaen,Aalborg University,http://people.cs.aau.dk/~ivan/,-8-9oRgAAAAJ
+Kristian Torp,Aalborg University,http://people.cs.aau.dk/~torp/,EusRZN0AAAAJ
+Kurt Nørmark,Aalborg University,http://people.cs.aau.dk/~normark/,5L7St-cAAAAJ
+Giorgio Bacci,Aalborg University,http://people.cs.aau.dk/~grbacci/,y2Q3kK8AAAAJ
+Anders Bruun,Aalborg University,http://personprofil.aau.dk/120220,X32A-C4AAAAJ
```
Comment (emeryberger) [inline csrankings.csv:16185]: I see a couple of people are not in CS but in "Institut for Datalogi". This doesn't sound like a CS department. Please explain how they meet the inclusion criteria. Thanks. A pointer to a web page from the CS department listing their faculty and affiliated faculty would be helpful.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 2 above)
Comment (hujilin1229) [reply csrankings.csv:16185]: "Institut for Datalogi" is Danish and it is translated to CS department in English. Here is a link to CS 
https://www.cs.aau.dk/research/. In this link, you can find the staff members by click into different groups on this paper, e.g., DATABASE, PROGRAMMING AND WEB TECHNOLOGIES (DPW). Hope it is enough.

Comment (csj42) [reply csrankings.csv:16185]: In Denmark, "Department of Computer Science" is called "Institut for Datalogi". This also applies to the Danish departments already in csrankings. Here is a link to the department's website: https://www.cs.aau.dk/about/. This page describes the organization: https://www.cs.aau.dk/about/organisation/.  This one contains links to the three research groups, each of which lists the members of the group: https://www.cs.aau.dk/research/ (see a link “Staff” at the bottom of each of the three pages).

In Denmark, one needs to be a professor or an associate professor (lektor) to be the primary supervisor of a Ph.D. student. Thus, we only report staff in those two categories.

File: csrankings.csv (hunk 1 above)
Comment (hujilin1229) [reply csrankings.csv:16157]: The URL problem is addressed in the latest commit.

**Comment by emeryberger** [comment]
OK, you will need to resolve conflicts and I will merge.

**Review by emeryberger — APPROVED** [review]

**Comment by hujilin1229** [comment]
> OK, you will need to resolve conflicts and I will merge.

Thanks! Conflicts are resolved.

**Pull request merged**